h264/mbType.go

```diff
@@ -87,38 +89,42 @@ func MbTypeName(sliceType string, mbType int) string {
 	return sliceTypeName
 }
 
-func MbPartPredMode(data *SliceData, sliceType string, mbType, partition int) string {
-	modeName := "UnknownPartPredMode"
+// Errors used by MbPartPredMode.
+var (
+	errNaMode    = errors.New("no mode for given slice and mb type")
+	errPartition = errors.New("partition must be 0")
+)
+
+func MbPartPredMode(data *SliceData, sliceType string, mbType, partition int) (mbPartPredMode, error) {
```

Comment:
Please don't make exported functions return unexported types.

Reply:
The function was already exported, and thought changing it would have been out of scope of the PR.

Reply:
Sure, but the return type could have been exported.